tests/Kernel/LanguageSwitcherTest.php

```diff
@@ -42,6 +49,41 @@ public function testLanguageSwitcherLinkListRendering(): void {
         preg_replace('/[\x00-\x1F\x7F\xA0]/u', '', $actual)
       );
     }
+
+    // Set Hungarian as non-EU language to test the non-EU category.
```

Comment:
Don't use EU languages as non-EU language in test please. Install a new one that is not EU.